pkg/objects/binder.go

```diff
@@ -0,0 +1,96 @@
+package objects
+
+import (
+	"fmt"
+	"mime/multipart"
+	"reflect"
+	"strings"
+
+	echo "github.com/theopenlane/echox"
+)
+
+var (
+	typeMultipartFileHeader      = reflect.TypeOf((*multipart.FileHeader)(nil))
+	typeMultipartSliceFileHeader = reflect.TypeOf(([]*multipart.FileHeader)(nil))
+)
+
+// BindFunc is a function that binds the request body to the struct pointer
+type BindFunc func(echo.Context, interface{}) error
```

Comment:
@golanglemonade did you end up using this binder anywhere?

Reply:
No, want me to pull it out @matoszz ?

Reply:
removing